m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -684,48 +684,48 @@ m0t1fs_oostore_mode_basic()
 
 	for i in {a..z} {A..Z} ; do
 		for c in `seq 1 4095`;
-			do echo -n $i ;
+			do echo -n "$i" ;
 		done;
 		echo;
 	done > $SOURCE_TXT
 
-	mount_m0t1fs $MOTR_M0T1FS_MOUNT_DIR "oostore" || rc=1
+	mount_m0t1fs "$MOTR_M0T1FS_MOUNT_DIR" "oostore" || rc=1
 	mount | grep m0t1fs                                 || rc=1
-	cp -v $SOURCE_TXT $MOTR_M0T1FS_MOUNT_DIR/$fsname1   || rc=1
-	cat $MOTR_M0T1FS_MOUNT_DIR/$fsname1 > /tmp/$fsname1 || rc=1
+	cp -v $SOURCE_TXT "$MOTR_M0T1FS_MOUNT_DIR"/$fsname1   || rc=1
```

Comment:
```
cp -v $SOURCE_TXT "$MOTR_M0T1FS_MOUNT_DIR"/"$fsname1" || rc=1
```
Do we also need to double quote the `$fsname1`?

Reply:
done